### PR TITLE
Custom line spacing became smaller after converting PPTX to HTML5 & missing space between paragraphs: fix paddings and margins

### PR DESCRIPTION
**Pull request by polonair**

### CODE REVIEW

File: Aspose.Slides.WebExtensions/Helpers/TextHelper.cs [hunk 1]
```diff
@@ -270,7 +272,8 @@ public static string GetTextStyle<T>(IPortionFormatEffectiveData format, ITextFr
         public static string GetTextStyle<T>(IParagraphFormatEffectiveData format, TemplateContext<T> model)
         {
             string alignment = TextHelper.GetHorizontalAlignmentStyle(format.Alignment);
-            string lineSpacingStyle = TextHelper.GetLineSpacingStyle(format);
+            float fontHeight = (model.Object as Paragraph).Portions[0].PortionFormat.GetEffective().FontHeight;
```
Comment (yury-knml) [inline Aspose.Slides.WebExtensions/Helpers/TextHelper.cs:275]: @polonair ,

Why not to use direct case, i.e. "((Paragraph)model.Object)"? I mean, here we choosing between NRE and InvalidCast and the last one looks better imo

File: Aspose.Slides.WebExtensions/Templates/common/bullet.html [hunk 2]
```diff
@@ -18,12 +18,13 @@
     }
     else
     {
-        margins[2] -= 12;  // FIXME magic number (ideally it should be bullet width)
+        margins[2] = -4;
```
Comment (yury-knml) [inline Aspose.Slides.WebExtensions/Templates/common/bullet.html:21]: @polonair ,

Meaningful comment about magic number disappear, but -4 not looks less magic then -12.

File: Aspose.Slides.WebExtensions/Helpers/TextHelper.cs (hunk 1 above)
Comment (polonair) [reply Aspose.Slides.WebExtensions/Helpers/TextHelper.cs:275]: Yes, I agree with you.

Comment (polonair) [reply Aspose.Slides.WebExtensions/Helpers/TextHelper.cs]: [looks like it's better not to use generics in this case](https://github.com/aspose-slides/Aspose.Slides.WebExtensions/pull/7/commits/c2345af07c654ba57a1d652a81f449394bdc2a44)

File: Aspose.Slides.WebExtensions/Templates/common/bullet.html (hunk 2 above)
Comment (polonair) [reply Aspose.Slides.WebExtensions/Templates/common/bullet.html]: [remove magic constant, just clean spaces between spans](https://github.com/aspose-slides/Aspose.Slides.WebExtensions/pull/7/commits/d70bed1aaad8517743008342951aea11c62d77e6)

**Pull request merged**